installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -105,4 +106,34 @@ You can use dedicated or shared IFLs to assign sufficient compute resources. Res
 
 include::modules/install-sno-ibm-z.adoc[leveloffset=+2]
 
-include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
+
+[id="installing-sno-with-ibmpower"]
+== Installing {sno} with {ibmpowerProductName}
+
+Installing a single-node cluster on {ibmpowerProductName} requires user-provisioned installation using either the "Installing a cluster with {op-system-base} KVM on {ibmpowerProductName}" procedure.
```

Comment:
We say: ". . . using either the "Installing a cluster with {op-system-base} KVM on {ibmpowerProductName}" procedure," but there is no other alternative listed. 

it should be ". . . either X procedure or Y procedure." my guess is that its "Installing single-node OpenShift with IBM Power" ?

Reply:
@stevsmit 

I see it written as: 
- Installing single-node OpenShift using the Assisted Installer
- Installing single-node OpenShift on AWS
- Installing single-node OpenShift with IBM Z and IBM® LinuxONE
- Installing single-node OpenShift with IBM Power